internal/app/types/user.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import "github.com/globalsign/mgo/bson"
+
+type User struct {
+	ID       bson.ObjectId   `json:"_id" bson:"_id,omitempty"`
```

Comment:
please add more fields: created_at, updated_at into it to keep track history and other tables should too